airbyte_cdk/sources/streams/http/http_client.py

```diff
@@ -332,6 +340,10 @@ def _send(
 
         return response  # type: ignore # will either return a valid response of type requests.Response or raise an exception
 
+    def _evict_key(self, prepared_request: requests.PreparedRequest) -> None:
```

Comment:
Can we add a comment to explain what is the consequence of not cleaning this? It seems like it wasn't obvious since it took us a bit of time to investigate and find the issue

Reply:
Yes, I added.